src/auth/FormElement.ts

```diff
@@ -95,6 +95,15 @@ export default class GmfAuthForm extends GmfBaseElement {
       [hidden] {
         display: none !important;
       }
+      .login-spinner {
+        float: left;
+        margin-right: 20px;
+      }
+
+      i.fa-spin {
```

Comment:
Not?
```suggestion
      .login-spinner.custom-spinner-connect {
```
With direct conversion, we will get that...

Reply:
I guess we don't need any class to just do that now. It is encapsulated in the web-component.